publish-mqtt/Cargo.toml

```diff
@@ -17,3 +16,8 @@ rustls = "0.17"
 rustls-native-certs = "0.3"
 tokio = "0.2.22"
 dotenv = "0.15.0"
+bluez-generated = { path = "../bluez-generated" }
+dbus-tokio = "*"
```

Comment:
- [ ] turn all of these * dependencies into concrete versions, and sort dependencies alphabetically